src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1267,7 +1265,7 @@ void Transformations::MainSnippets() {
          ov::intel_cpu::brgemm_utils::is_fp16_supported());
     const bool isMHASupported = !is_LLM && is_infer_prc_supported_by_brgemm;
 #else
-    const bool isMHASupported = false;
+    const bool isMHASupported = any_of(config.inferencePrecision, ov::element::f32, ov::element::dynamic);
```

Comment:
1. The same concern as in [this discussion](https://github.com/openvinotoolkit/openvino/pull/31754#discussion_r2286304302)
2. Until we are not sure that everything is OK in real model scenarios, we can enable MHA tokenization only in tests (as it is done for MLP). So please take `ignoreCallback` value into account here
3. Let's create `is_infer_prc_supported_by_brgemm` local variable for ARM64 as well to reuse it for `isMHASupported` and  `isMlpSeqSupported`